### DISCUSSION TITLE
希儿什么时候能嫁给我

### DISCUSSION THREAD

**Discussion by lyc736068585** [share121/inter-knot#2421] [其他]
![113078686_p0_master1200](https://github.com/user-attachments/assets/6a0bd2b7-a072-4b49-a060-b7f3fcd12949)
<img src="https://img-static.mihoyo.com/communityweb/upload/43e9d24d154f4da394497f49ee3dfa11.png" alt="希儿比心1" width="75" ><img src="https://img-static.mihoyo.com/communityweb/upload/95f33cbdff7e4182cf26cec4415cb71d.png" alt="希儿比心2" width="75" >

**Comment by Genshin-Neuvillette** [comment]
等星见雅嫁给我就可以了

**Comment by YuTinMIn** [comment]
答：V我648，她就会嫁给你了。

**Comment by wtnl551** [comment]
你什么时候睡，希儿什么时候嫁给你

**Comment by cpu644** [comment]
阿道夫-希儿？

**Comment by BitterWind** [comment]
不知道,(摸摸希儿脑袋),等她睡醒了我问问她.

**Comment by Bai6869** [comment]
睡觉就行

**Comment by nina-mmk** [comment]
想桃子

**Comment by Kujojolen** [comment]
德国的希儿？

**Comment by HarukaTea** [comment]
图先偷了
